app/controllers/oaisys/pmh_controller.rb

```diff
@@ -2,25 +2,84 @@
 
 module Oaisys
   class PMHController < ApplicationController
+    EXPECTED_ARGUMENTS = {
+        identify: {
+            required: %w(),
+            optional: %w(),
+            exclusive: %w()
+        },
+        list_sets: {
+            required: %w(),
+            optional: %w(),
+            exclusive: %w(resumptionToken)
```

Comment:
What's the idea behind exclusive? I understand the other two categories.

Reply:
An exclusive argument means "the argument may be included with request, but must be the only argument (in addition to the verb argument)." From what I can tell our currently oai implementation doesn't do a check to ensure that exclusive arguments are the only arguments.

Reply:
Ah, gotcha. Thanks!